freetextresponse/freetextresponse.py

```diff
@@ -294,7 +294,7 @@ def student_view(self, context={}):
                 'visibility_class': self._get_indicator_visibility_class(),
                 'word_count_message': self._get_word_count_message(),
                 'display_other_responses': self.display_other_student_responses,
-                'other_responses': [],
+                'other_responses': self.get_other_answers(self.get_student_id()),
```

Comment:
Since we're never re-using a memoized version of this, just move the `get_student_id` lookup into the method itself.